Employees/Controllers/DeveloperController.cs

```diff
@@ -0,0 +1,31 @@
+using Employees.Models;
+using Microsoft.AspNetCore.Mvc;
+
+
+namespace Employees.Controllers
+{
+    [Route("api/developer")]
+    [Produces("application/json")]
+    [ApiController]
+    public class DeveloperController : ControllerBase
+    {
+        [HttpPost("set-manager")]
+        public LeadDeveloper SetManager([FromBody] LeadDeveloper leadDeveloper)
+        {
+            Developer developer = new Developer("Qais", "Maher", 500);
```

Comment:
add static class to set all the employees info.
here you only need to pass the employees